### PR TITLE
feat(ci): add cypress test ui based ingestion

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        cy.clickOptionWithText("Skip")
+
+        cy.enterTextInTestId('source-name-input', testName)
+        cy.clickOptionWithText("Advanced")
+        cy.enterTextInTestId('cli-version-input', "0.9.3.3rc5")
+        cy.clickOptionWithText("Save & Run")
+        cy.waitTextVisible(testName)
+
+        // managed ingestion takes few seconds
+        cy.wait(10000)
```
Comment (aditya-radhakrishnan) [inline smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js:34]: do we really need to wait 10 seconds??

File: smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        readyToTypeEditor().type("{enter}");
+        // no space because the editor starts new line at same indentation
+        readyToTypeEditor().type("config: {}");
+        cy.clickOptionWithText("Next")
+
+        cy.clickOptionWithText("Skip")
+
+        cy.enterTextInTestId('source-name-input', testName)
+        cy.clickOptionWithText("Advanced")
+        cy.enterTextInTestId('cli-version-input', "0.9.3.3rc5")
```
Comment (aditya-radhakrishnan) [inline smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js:29]: can we pull this out into a constant?

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM besides for a couple small things

**Comment by anshbansal** [comment]
@aditya-radhakrishnan Will take care of the review comments. This should not be merged yet because this still needs some changes as the test is failing on github when other tests are also being run.

File: smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js (hunk 2 above)
Comment (anshbansal) [reply smoke-test/tests/cypress/cypress/integration/mutations/managed_ingestion.js:29]: will do

**Pull request merged**